v3/src/components/graph/adornments/movable-point/movable-point-types.ts

```diff
@@ -2,4 +2,3 @@ export const kMovablePointClass = "movable-point"
 export const kMovablePointType = "Movable Point"
 export const kMovablePointPrefix = "ADRN"
 export const kMovablePointLabelKey = "DG.Inspector.graphMovablePoint"
-export const kMovablePointHint = ""
```

Comment:
I meant to remove this unused variable as part of the [Add Movable Point PR](https://github.com/concord-consortium/codap/pull/838) but missed it. This change doesn't affect the fade in/out at all.